src/components/TimePicker.tsx

```diff
@@ -36,18 +37,21 @@ const DateTimePickerBase: React.FC<DateTimePickerBase> = ({
   }, [])
 
   return (
-    <MobileDateTimePicker
-      {...restProps}
-      dateAdapter={memoizedDateAdapter}
-      disablePast
-      minutesStep={BATCH_TIME / 60}
-      // inputFormat="yyyy/MM/dd HH:mm"
-      ampm={false}
-      minDateTime={minDateTime}
-      renderInput={(props): JSX.Element => (
-        <TextField {...props} label="Set custom date" name={inputName} error={!!error} />
-      )}
-    />
+    <>
+      <GlobalStyles />
+      <MobileDateTimePicker
```

Comment:
Btw, you can pass `className` to `MobileDateTimePicker`, which means can wrap it in `styled(MobileDateTimePicker)`.
That class gets applied to `.MuiPickersBasePicker-container`.
Will it be enough to cover all the global styles? Probably not

Reply:
> Will it be enough to cover all the global styles? Probably not

:joy: :point_up: my fear